<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventEditorForm</class>
 <widget class="QWidget" name="EventEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Event relation information</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="eventRoleLabel">
            <property name="text">
             <string>Rol</string>
            </property>
            <property name="buddy">
             <cstring>eventRoleComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="eventRoleComboBox"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Event information</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="eventTypeLabel">
            <property name="text">
             <string>Soort gebeurtenis</string>
            </property>
            <property name="buddy">
             <cstring>eventTypeComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="eventTypeComboBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="eventDateLabel">
            <property name="text">
             <string>Datum</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
            <item>
             <widget class="QLineEdit" name="eventDatePicker"/>
            </item>
            <item>
             <widget class="QPushButton" name="eventDateEditButton">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset theme="edit-entry"/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="eventNameLabel">
            <property name="text">
             <string>Beschrijving</string>
            </property>
            <property name="buddy">
             <cstring>eventNameEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="eventNameEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="dialogButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
